pkg/controller/windowsmachine/windows/windows.go

```diff
@@ -228,7 +234,10 @@ func (vm *windows) Run(cmd string, psCmd bool) (string, error) {
 
 	out, err := vm.interact.run(cmd)
 	if err != nil {
-		log.Error(err, "error running", "cmd", cmd, "out", out)
+		// Hack to not print the error log for "sc.exe qc" returning 1060 for non existent services.
+		if !(strings.HasPrefix(cmd, serviceQueryCmd) && strings.HasSuffix(err.Error(), serviceNotFound)) {
+			log.Error(err, "error running", "cmd", cmd, "out", out)
+		}
```

Comment:
I'm worried about putting special cases like this in Run(), in the future we might have more cases like that. Did you consider not logging within `Run()` but logging at the callsite if the error is actually an error?

Reply:
Yes I did. This means we have to keep ensuring folks log at the call site for new code. I prefer this for now and we can revisit if we keep adding exceptions.

Reply:
Sounds good to me